File: WebUI/UI/src/app/datas/models/question-types/rating-question.ts
export class RatingQuestion {
  constructor(questionId: number, isRequired: boolean, rate: number) {
    this.questionId = questionId;
    this.isRequired = isRequired;
    this.rate = rate;
  }

  questionId: number;
  isRequired: boolean;
  rate: number;
}

File: Entities/QuestionTypes/RatingQuestion.cs
using System.ComponentModel.DataAnnotations;
using Core.Entities;

namespace Entities.QuestionTypes
{
    public class RatingQuestion : IEntity
    {
        [Key]
        public int QuestionId { get; set; }
        public bool IsRequired { get; set; }
        public int Rate { get; set; }
    }
}
